### ISSUE TITLE
feat(YouTube/`Hide Shopping Links in Video Description`:  Hide the shopping links provided in the description 

### ISSUE THREAD

**Issue by ILoveOpenSourceApplications** [YT-Advanced/ReX-patches#117]
### Application

YouTube

### Feature description

The litho component `expandable_list_inner` can be used to hide the shopping links provided by the channels in their video descriptions. 
Link for reference:
https://youtu.be/_euc0Hf_Vp4

### Motivation

Hides the ad links of products and can come under ads settings. 

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by YT-Advanced** [comment]
Can you share a screenshot of this ads

**Comment by ILoveOpenSourceApplications** [comment]
`expandable_list_inner` hides product ads from the description.
Before:
![Screenshot_20230611-125646~2](https://github.com/revanced/revanced-patches/assets/117499019/1040a30a-dc90-49d6-82e2-a00520ccd55b)
After:
![Screenshot_20230611-130317~2](https://github.com/revanced/revanced-patches/assets/117499019/c44901b2-6eca-4179-b488-448b589eefee)

**Comment by YT-Advanced** [comment]
Thanks

**Comment by ILoveOpenSourceApplications** [comment]
I see that the litho component got added into general ads patch but there's no separate toggle for it. I tried to turn off various ads button and none of it brought back the ad until I restarted the app. Which made me do the same about other ad toggles and some of them looks like it requires an app restart to function. Should I open a separate issue for this or can it be squashed with this one itself?
